doc/platform.txt

```diff
@@ -5,7 +5,10 @@ Synopsis
 Manage project platforms
 
     add <plat-spec> [...].............. add specified platforms
-        --usegit ...................... retrieve from git instead of npm registry
+        --usegit ...................... [WARNING: This flag has been deprecated !]
+                                        [Instead, please use: `cordova platform add git-url#custom-branch`.] 
+                                        [e.g: cordova platform add https://github.com/apache/cordova-android.git#2.4.0]
+                                        retrieve from git instead of npm registry
```

Comment:
Just a minor nit, as for the other change - don't put a space before the `!` (or just replace it with a period). Also, I wonder - should the description come first, then the warning?